src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -30,7 +30,9 @@ internal class OpsConfigAdapter : IDisposable
 
         public static readonly DocsEnvironment DocsEnvironment = GetDocsEnvironment();
 
-        private static readonly SecretClient s_secretClient = new SecretClient(new Uri("https://docfx.vault.azure.net"), new DefaultAzureCredential());
+        private static readonly Lazy<SecretClient> s_secretClient = new Lazy<SecretClient>(
```

Comment:
These two should be both thread safe, only initialized once.

Reply:
now should be the right fix.